test/controllers/trust.information.controller.spec.ts

```diff
@@ -73,13 +72,5 @@ describe("TRUST INFORMATION controller", () => {
       expect(resp.status).toEqual(302);
       expect(resp.header.location).toEqual(config.TRUST_INFO_URL);
     });
-
-    test("catch error when rendering the page", async () => {
```

Comment:
Wondering why this test has been removed - surely it's still a valid test?

You need to add a test for when trust data box is empty though now, to check the new validation implementation you've put in place. Plenty of examples of doing that in other controllers.

Reply:
Probably something like this will do
```
const resp = await request(app)
   .post(config.TRUST_INFO_URL);
   .send({TrustKey: "Something" });
```

Reply:
Yep, I removed it as it wasn't testing the controller code. After adding validation tests, I've managed to get it working again (thanks Mo!)